src/toil/resource.py

```diff
@@ -243,15 +258,15 @@ class FileResource(Resource):
     """A resource read from a file on the leader."""
 
     @classmethod
-    def _load(cls, path):
+    def _load(cls, path: str) -> TextIO:
         return open(path)
```

Comment:
If we harmonize on binary-mode streams, this would change to a binary mode open and a binary-mode return value.